socialhome/users/views.py

```diff
@@ -69,8 +96,14 @@ def get_context_data(self, **kwargs):
         context["pinned_content_exists"] = True
         context["stream_name"] = "profile__%s" % self.object.id
         context["profile_stream_type"] = "pinned"
+
+        context["json_context"] = self._get_json_context(context)
```

Comment:
This could be in `ProfileViewMixin`?

Reply:
Nope. Needs `context["stream_name"` and `context["pinned_content_exists"]` to be set.